*** 2017-02-07 16:50:48.23 idempiere Log (CLogFile) ***
16:50:48.229 Adempiere.startup: iDempiere(r) Release 3.1_2015-10-31 -Smart Suite ERP, CRM and SCM- (c) 1999-2013 iDempiere(r); Implementation: iDempiere - Supported by iDempiere community [17]
16:50:48.233 Ini.loadProperties: E:\mercurialidempiere/idempiere-dev.properties #33 [17]
16:50:48.898 DB.isBuildOK: Build DB=${env.ADEMPIERE_VERSION} 20080428-1232 [17]
16:50:48.898 DB.isBuildOK: Build Cl=3.1.0.qualifier [17]
16:50:48.936-----------> DB.isBuildOK: Build Version Error 

The program assumes build version 3.1.0.qualifier, but database has build version ${env.ADEMPIERE_VERSION} 20080428-1232. 
This is likely to cause hard to fix errors. 
Please contact administrator. [17]
16:50:49.105 WebEnv.initWeb: Servlet Context Init Parameters: iDempiere Root
ServerRoot=1 [17]
16:50:49.296-----------> EMail.setFrom: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
16:50:49.302-----------> EMail.addTo: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
16:50:49.302-----------> EMail.isValid: From is invalid=null [17]
16:50:49.302-----------> EMail.isValid: From is invalid=null [17]
16:50:49.303-----------> MClient.sendEMail: Could NOT Send Email: Server started: iDempiere (odbz:192.168.47.1) to you @ company.org: Invalid Data (System) [17]
16:50:49.303 WebEnv.initWeb: Servlet Init Parameter: idempiereMonitor [17]
16:50:49.303 AdempiereMonitor.init:  [17]
16:50:51.471-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]
16:50:51.733-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://180.fwk1480207683:2/metainfo/zk/config.xml
with bundleresource://180.fwk1480207683:19/metainfo/zk/config.xml [17]
16:50:51.739-----------> ClassLocator.getDependentXMLResources: Replicate resource: zk
Overwrite bundleresource://180.fwk1480207683:11/metainfo/zk/config.xml
with bundleresource://180.fwk1480207683:28/metainfo/zk/config.xml [17]
16:50:51.745-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://180.fwk1480207683:12/metainfo/zk/config.xml
with bundleresource://180.fwk1480207683:29/metainfo/zk/config.xml [17]
16:50:51.775-----------> ClassLocator.getDependentXMLResources: Replicate resource: zul
Overwrite bundleresource://180.fwk1480207683:14/metainfo/zk/zk.xml
with bundleresource://180.fwk1480207683:31/metainfo/zk/zk.xml [17]
16:50:52.204-----------> LanguageDefinition.<init>: Replicated language: xhtml, overriden by [LanguageDefinition: xhtml] [17]
16:50:52.230-----------> LanguageDefinition.<init>: Replicated language: xul/html, overriden by [LanguageDefinition: xul/html] [17]
16:50:52.296-----------> ClassLocator.getDependentXMLResources: Replicate resource: calendar
Overwrite bundleresource://180.fwk1480207683:1/metainfo/zk/lang-addon.xml
with bundleresource://180.fwk1480207683:18/metainfo/zk/lang-addon.xml [17]
16:50:52.299-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://180.fwk1480207683:2/metainfo/zk/lang-addon.xml
with bundleresource://180.fwk1480207683:19/metainfo/zk/lang-addon.xml [17]
16:50:52.304-----------> ClassLocator.getDependentXMLResources: Replicate resource: gmapsz
Overwrite bundleresource://180.fwk1480207683:5/metainfo/zk/lang-addon.xml
with bundleresource://180.fwk1480207683:22/metainfo/zk/lang-addon.xml [17]
16:50:52.311-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeline
Overwrite bundleresource://180.fwk1480207683:6/metainfo/zk/lang-addon.xml
with bundleresource://180.fwk1480207683:23/metainfo/zk/lang-addon.xml [17]
16:50:52.316-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeplot
Overwrite bundleresource://180.fwk1480207683:7/metainfo/zk/lang-addon.xml
with bundleresource://180.fwk1480207683:24/metainfo/zk/lang-addon.xml [17]
16:50:52.330-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://180.fwk1480207683:12/metainfo/zk/lang-addon.xml
with bundleresource://180.fwk1480207683:29/metainfo/zk/lang-addon.xml [17]
16:50:52.339-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkplus
Overwrite bundleresource://180.fwk1480207683:13/metainfo/zk/lang-addon.xml
with bundleresource://180.fwk1480207683:30/metainfo/zk/lang-addon.xml [17]
16:50:52.622           WebUIServlet.init: iDempiere Web Client started successfully [17]
16:50:53.722-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]
16:50:59.684-----------> LCO_ValidatorDN.initialize:  [70]
16:50:59.731-----------> LCO_ValidatorWH.initialize:  [70]

*** 2017-02-07 16:50:59.98 idempiere Log (CLogFile) ***
16:51:02.753-----------> Node: [192.168.163.1]:5701 [adempiere.idempiere/localhost] [3.5.3] No join method is enabled! Starting standalone. [50]
16:53:19.843===========> TableIndexElementHandler.validateTableIndex: ERROR: no existe la columna am.amcanorder
  Position: 427 [71]
org.postgresql.util.PSQLException: ERROR: no existe la columna am.amcanorder
  Position: 427; State=42703; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
	at org.postgresql.jdbc2.AbstractJdbc2DatabaseMetaData.getIndexInfo(AbstractJdbc2DatabaseMetaData.java:4230)
	at com.mchange.v2.c3p0.impl.NewProxyDatabaseMetaData.getIndexInfo(NewProxyDatabaseMetaData.java:3328)
	at org.compiere.process.TableIndexValidate.validateTableIndex(TableIndexValidate.java:69)
	at org.adempiere.pipo2.handler.TableIndexElementHandler.validateTableIndex(TableIndexElementHandler.java:112)
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:93)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:260)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:253)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:325)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)

16:54:12.457===========> PackIn.importXML: importXML: [71]
org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:260)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:253)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:325)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

16:54:12.530===========> PackIn.importXML: importXML: [71]
java.lang.RuntimeException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:188)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)
	at org.adempiere.plugin.utils.Incremental2PackActivator.installPackage(Incremental2PackActivator.java:120)
	at org.adempiere.plugin.utils.Incremental2PackActivator.access$4(Incremental2PackActivator.java:89)
	at org.adempiere.plugin.utils.Incremental2PackActivator$2.run(Incremental2PackActivator.java:259)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
caused by: org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:260)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:253)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:325)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

16:54:12.576===========> PipoDictionaryService.merge: importXML: [71]
java.lang.RuntimeException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:188)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)
	at org.adempiere.plugin.utils.Incremental2PackActivator.installPackage(Incremental2PackActivator.java:120)
	at org.adempiere.plugin.utils.Incremental2PackActivator.access$4(Incremental2PackActivator.java:89)
	at org.adempiere.plugin.utils.Incremental2PackActivator$2.run(Incremental2PackActivator.java:259)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
caused by: org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:260)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:253)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:325)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

